.github/workflows/php-tests.yml

```diff
@@ -55,11 +55,12 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
-          coverage:    none
+          coverage:    xdebug
 
       - name: If PHP >= 8.0 set up PHPUnit 9.5 for compatibility
         if: ${{ matrix.php >= '8.0' }}
-        run: wget https://phar.phpunit.de/phpunit-9.5.13.phar && mv phpunit-9.5.13.phar phpunit.phar
-
+        run: wget https://phar.phpunit.de/phpunit-9.5.28.phar && mv phpunit-9.5.28.phar phpunit.phar
```

Comment:
The previous version (`9.5.13`) had a bug when generating the coverage report. The generated XML file had no labels for the file list, so it was not possible to identify which file had which %.